docs/releases/3.1.txt

```diff
@@ -433,6 +433,8 @@ Miscellaneous
 * The :setting:`TIME_ZONE <DATABASE-TIME_ZONE>` setting is now allowed on
   databases that support time zones.
 
+* Added a system check to enforce async-safety environment on deployment
```

Comment:
One for the fellows to look at later - I think we could do with an "async" section in the release notes from now on?

Also this sentence should end with a full stop.

Reply:
> Also this sentence should end with a full stop.

Oh sorry, my bad.